vbox/docker-entrypoint.sh

```diff
@@ -114,16 +124,16 @@ if [ "$1" = 'daemon' ]; then
   cd /cuckoo
   rm -rf pidfiles/*.pid
 
-  exec cuckoo -d "$@"
+  su-exec cuckoo cuckoo -d "$@"
```

Comment:
so su-exec cuckoo is kinda the same as setuid it steps down from root and runs as cuckoo

Reply:
The problems is that uid is changed in entrypoint, but we still want to run cuckoo as `cuckoo` user.